Core/PARStore.m

```diff
@@ -1458,8 +1468,11 @@ - (void)_sync
     
     // just go through each row (back in time) until all entries are loaded
     NSMutableDictionary *updatedValues = [NSMutableDictionary dictionary];
-    for (NSManagedObject *log in allLogs)
+    for (NSManagedObjectID *logID in allLogObjectIDs)
     {
+        // Get object
+        NSManagedObject *log = [moc objectWithID:logID];
```

Comment:
Correct indeed, controllable by `includesPropertyValues` if you know you don't want this to happen